packages/playwright-core/src/client/network.ts

```diff
@@ -141,7 +141,7 @@ export class Request extends ChannelOwner<channels.RequestChannel> implements ap
       return null;
 
     const contentType = this.headers()['content-type'];
-    if (contentType.includes('application/x-www-form-urlencoded')) {
+    if (contentType?.includes('application/x-www-form-urlencoded')) {
```

Comment:
nit: to be even more accurate, I'd split the content type by `;` and then check if there is the mime type.